Source/ThirdParty/libwebrtc/Source/webrtc/webkit_sdk/WebKit/CMBaseObjectSPI.h

```diff
@@ -47,6 +49,10 @@ enum {
     kCMBaseObjectError_ValueNotAvailable = -12783,
 };
 
+enum {
+    kCMBaseObjectDescriptionOffset_NotPresent = 0,
+};
```

Comment:
I need to put this somewhere rather than hard-coding `0` in the patch below.

Reply:
Doh.  I didn't realize `CMBaseObjectSPI.h` was marked as a _public_ header for the `libwebrtc` project.

Is that correct?

Reply:
That is the way it is done to allow including this header in WebCore.